src/OSPSuite.Core/Converters/v12/Converter110To120.cs

```diff
@@ -33,7 +35,64 @@ public Converter110To120(IObjectPathFactory objectPathFactory)
 
       public (int convertedToVersion, bool conversionHappened) ConvertXml(XElement element)
       {
-         return (PKMLVersion.V12_0, false);
+         _converted = false;
+         element.DescendantsAndSelfNamed("Simulation").Each(convertSimulation);
+         return (PKMLVersion.V12_0, _converted);
+      }
+
+      private void convertSimulation(XElement simulationElement)
+      {
+         //use to list because the node will be deleted
+         simulationElement.Descendants("BuildConfiguration").ToList().Each(convertBuildConfigurationToSimulationConfiguration);
+      }
+
+      private void convertBuildConfigurationToSimulationConfiguration(XElement buildConfigurationElement)
```

Comment:
@rwmcintosh @georgeDaskalakis 
A little bit nasty but I am converting one xml structure to another BEFORE the serializer is kicking in. While this structure is temporary, we can do sthg like this to ensure our old pkml are valid (we'll have to potentially create another node such as ModuleList or sthg like this but this is it)

Reply:
Also we do not need to define any constants for those string IMO. This is only use for project conversion and if we change one, the tests will fail